Docker/docker-compose.yml

```diff
@@ -9,77 +9,47 @@ services:
     networks:
       - dalgo-network 
   backend:
-    image: dalgo_backend:0.1
+    image: dalgo_backend:latest
+    command: backend
     restart: always
     ports:
       - "8002:8002"
-    environment:
-      - DJANGOSECRET=${DJANGOSECRET}
-      - DEBUG=${DEBUG}
-      - PRODUCTION=${PRODUCTION}
-      - DEV_SECRETS_DIR=${DEV_SECRETS_DIR}
-      - USE_AWS_SECRETS_MANAGER=${USE_AWS_SECRETS_MANAGER}
-      - DBNAME=${DBNAME}
-      - DBHOST=${DBHOST}
-      - DBPORT=${DBPORT}
-      - DBUSER=${DBUSER}
-      - DBPASSWORD=${DBPASSWORD}
-      - DBADMINUSER=${DBADMINUSER}
-      - DBADMINPASSWORD=${DBADMINPASSWORD}
-      - REDIS_HOST=${REDIS_HOST}
-      - REDIS_PORT=${REDIS_PORT}
-      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
-      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
-      - AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION}
-      - SES_ACCESS_KEY_ID=${SES_ACCESS_KEY_ID}
-      - SES_SECRET_ACCESS_KEY=${SES_SECRET_ACCESS_KEY}
-      - SES_SENDER_EMAIL=${SES_SENDER_EMAIL}
-      - AIRBYTE_SERVER_HOST=${AIRBYTE_SERVER_HOST}
-      - AIRBYTE_SERVER_PORT=${AIRBYTE_SERVER_PORT}
-      - AIRBYTE_SERVER_APIVER=${AIRBYTE_SERVER_APIVER}
-      - AIRBYTE_API_TOKEN=${AIRBYTE_API_TOKEN}
-      - AIRBYTE_DESTINATION_TYPES=${AIRBYTE_DESTINATION_TYPES}
-      - PREFECT_PROXY_API_URL=${PREFECT_PROXY_API_URL}
-      - PREFECT_HTTP_TIMEOUT=${PREFECT_HTTP_TIMEOUT}
-      - CLIENTDBT_ROOT=${CLIENTDBT_ROOT}
-      - DBT_VENV=${DBT_VENV}
-      - SIGNUPCODE=${SIGNUPCODE}
-      - CREATEORG_CODE=${CREATEORG_CODE}
-      - FRONTEND_URL=${FRONTEND_URL}
-      - SENDGRID_APIKEY=${SENDGRID_APIKEY}
-      - SENDGRID_SENDER=${SENDGRID_SENDER}
-      - SENDGRID_RESET_PASSWORD_TEMPLATE=${SENDGRID_RESET_PASSWORD_TEMPLATE}
-      - SENDGRID_SIGNUP_TEMPLATE=${SENDGRID_SIGNUP_TEMPLATE}
-      - SENDGRID_INVITE_USER_TEMPLATE=${SENDGRID_INVITE_USER_TEMPLATE}
-      - SENDGRID_YOUVE_BEEN_ADDED_TEMPLATE=${SENDGRID_YOUVE_BEEN_ADDED_TEMPLATE}
-      - PREFECT_NOTIFICATIONS_WEBHOOK_KEY=${PREFECT_NOTIFICATIONS_WEBHOOK_KEY}
-      - SUPERSET_USAGE_DASHBOARD_API_URL=${SUPERSET_USAGE_DASHBOARD_API_URL}
-      - SUPERSET_USAGE_CREDS_SECRET_ID=${SUPERSET_USAGE_CREDS_SECRET_ID}
-      - FIRST_ORG_NAME=${FIRST_ORG_NAME}
-      - FIRST_USER_EMAIL=${FIRST_USER_EMAIL}
-      - ADMIN_USER_EMAIL=${ADMIN_USER_EMAIL}
-      - ADMIN_USER_PASSWORD=${ADMIN_USER_PASSWORD}
-      - DEMO_SIGNUPCODE=${DEMO_SIGNUPCODE}
-      - DEMO_AIRBYTE_SOURCE_TYPES=${DEMO_AIRBYTE_SOURCE_TYPES}
-      - DEMO_SENDGRID_SIGNUP_TEMPLATE=${DEMO_SENDGRID_SIGNUP_TEMPLATE}
-      - DEMO_SUPERSET_USERNAME=${DEMO_SUPERSET_USERNAME}
-      - DEMO_SUPERSET_PASSWORD=${DEMO_SUPERSET_PASSWORD}
+    env_file:
+      - .env.docker
+    volumes:
+      - ${CLIENTS_DBT_MOUNT}:/data/clients_dbt
+      - ${DEV_SECRETS_MOUNT}:/data/secrets
+      - ${LOGS_MOUNT}:/usr/src/backend/ddpui/logs
```

Comment:
`/usr/src/backend/ddpui` ... what is this

Reply:
its the `HOME_DIR`, defined at the top. 
I am not fussy about it, it can also be `/` or `/app` or anything. Its where we copy our code to.